src/main/java/com/jfrog/ide/idea/scan/data/Driver.java

```diff
@@ -48,4 +48,12 @@ public boolean equals(Object other) {
         return (Objects.equals(this.name, rhs.name) && (CollectionUtils.isEqualCollection(this.rules, rhs.rules)));
     }
 
+    public Rule getRuleById(String ruleId) {
```

Comment:
please add a test to it, and embed a test for it in: `parseOutputSarif` - where this error is the expected result

Reply:
i am unsure where to put the test honestly

Reply:
check again. was added to sarif.